src/libraries/System.Private.CoreLib/src/System/MemoryExtensions.cs

```diff
@@ -401,6 +396,192 @@ public static ReadOnlyMemory<char> AsMemory(this string? text, Range range)
             return SpanHelpers.Contains(ref MemoryMarshal.GetReference(span), value, span.Length);
         }
 
+        /// <inheritdoc cref="ContainsAny{T}(ReadOnlySpan{T}, T, T)"/>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Are all of these AggressiveInlinings actually needed?

Reply:
@EgorBo, what should our approach here be in general?

Reply:
I'd be surprised to see it non-inlined, although, this will never be inlined even with the Aggressive attribute for shared T due to runtime lookup

Reply:
Do we want to be in the habit of annotating everything as AggressiveInlining we want inlined even if should be anyway?

Reply:
> Do we want to be in the habit of annotating everything as AggressiveInlining we want inlined even if should be anyway?

I am not a fan of that approach as it _offends_ the jit 😄 although, we have other runtimes. I can probably run some script to find methods <= 16 bytes of IL with AggressiveInlining just out of curiosity (JIT always inlines methods <= 16 bytes of IL so the attribute is basically no-op)